src/geometry/manifold/partition_of_unity.lean

```diff
@@ -6,69 +6,83 @@ Authors: Yury G. Kudryashov
 import topology.paracompact
 import topology.shrinking_lemma
 import geometry.manifold.bump_function
+import topology.partition_of_unity
 
 /-!
 # Smooth partition of unity
 
-In this file we define `smooth_bump_covering`, a structure that will be used to construct a smooth
-partition of unity. Namely, a `smooth_bump_covering` of a set `s : set M` is a collection of
-`smooth_bump_function`s such that their supports is a locally finite family of sets, and for each
-point `x ∈ s` there exists a bump function `f i` in the collection such that `f i =ᶠ[𝓝 x] 1`. This
-structure is the main building block in the construction of a smooth partition of unity (see TODO),
-and can be used instead of a partition of unity in some proofs.
+In this file we define two structures, `smooth_bump_covering` and `smooth_partition_of_unity`. Both
+structures describe coverings of a set by a locally finite family of supports of smooth functions
+with some additional properties. The former structure is mostly useful as an intermediate step in
+the construction of a smooth partition of unity but some proofs that traditionally deal with a
+partition of unity can use a `smooth_bump_covering` as well.
 
-We say that `f : smooth_bump_covering I s` is *subordinate* to a map `U : M → set M` if for each
+Given a real manifold `M` and its subset `s`, a `smooth_bump_covering I M s` is a collection of
+`smooth_bump_function`s `f i` indexed by `i : f.ι` such that
+
+* the center of each `f i` belongs to `s`;
+* the family of sets `support (f i)` is locally finite;
+* for each `x ∈ s`, there exists `i : f.ι` such that `f i =ᶠ[𝓝 x] 1`.
```

Comment:
You forgot to change these lines when you put iota as a parameter instead of a structure member.